services/web-app/secure-dev-server.js

```diff
@@ -11,9 +11,9 @@
 const { createServer } = require('https')
 const next = require('next')
 const fs = require('fs')
-const { RunMode, getRunMode, loadEnvVars } = require('@carbon-platform/api/runtime')
+const { RunMode, getRunMode } = require('@carbon-platform/api/runtime')
 
-const { PORT } = loadEnvVars({ PORT: '443' })
+const PORT = 3001
```

Comment:
Any chance for this file that we can remove it and combine it with the new proxy-server file?

https://github.com/http-party/node-http-proxy#using-https

(of particular interest is the [https->http](https://github.com/http-party/node-http-proxy#https---http) section.

These docs seem to indicate that the express app can bear the responsibility for TLS termination, leaving the web-app (proxy target) as-is using http.

Definitely check me on that, but if we can do that, I think it would go a long way toward simplifying the logic.

Reply:
Done!!